### DISCUSSION TITLE
Duplicate routes found

### DISCUSSION THREAD

**Discussion by luyizhang** [facebook/docusaurus#7936] [Q&A]
I'm building a blog-only site with docusaurus and followed the instructions here: [https://docusaurus.io/docs/blog](url) (under blog-only mode). I removed the `index.js` under `./src/pages/index.js`. 

However, when I try to run the dev server, it gives a warning about duplicate routes at `/` that could lead to non-deterministic routing behavior. The site will build properly, but will have nothing on the site and all links are broken.

If I do a local build with `npm run build and `npm run serve`, the site will work properly (but I'll still get the same warning).

I'm not sure where the issue could be as my `src/pages/` folder is totally empty. I also deleted the docs folder.

This is what my config and presets look like: 
`const config = {
  title: "Blog",
  tagline: 'tbd',
  url: 'https://www.google.com',
  baseUrl: '/',
  onBrokenLinks: 'throw',
  onBrokenMarkdownLinks: 'warn',
  favicon: 'img/favicon.ico',
`
`presets: [
    [
      '@docusaurus/preset-classic',
      {
        docs: false,
        blog: {
          id: 'blog',
          routeBasePath: '/',
          showReadingTime: true,
          blogSidebarTitle: "All Posts",
          blogSidebarCount: "ALL"
        },
        theme: {
          customCss: require.resolve('./src/css/custom.css'),
        },
      },
    ],
  ],`

Any help would be much appreciated!

**Comment by slorber** [comment]
Please create a repro, we can't help otherwise

**Reply by luyizhang** [reply]
Here's a repo: https://github.com/luyizhang/test-docusaurus.

I'm using Chrome as a browser, and it won't show anything when I run locally. It'll show if I use incognito though. This repo also doesn't have the `blogTitle | siteTitle` working.

Thanks for your help!

**Reply by slorber** [reply]
It works for me and I don't get any duplicate route warning on "start" or "build"

https://stackblitz.com/github/luyizhang/test-docusaurus/

<img width="574" alt="CleanShot 2022-08-16 at 19 40 43@2x" src="https://user-images.githubusercontent.com/749374/184944155-1679d4a9-f2c0-4e08-8a8f-f72c0f5eb888.png">

Now I admin the title thing is a bit misleading.

```
  const title = isBlogOnlyMode ? siteTitle : blogTitle;
```

In blogOnly mode, we use the site title instead of the blog title.

And there's this annoying legacy bug that duplicates to "SiteTitle | SiteTitle", see also https://github.com/facebook/docusaurus/issues/5878 and referenced issues

**Reply by luyizhang** [reply]
The duplicate routes issue went away, but now I'm having trouble with getting the site to render locally in the Chrome browser.

When I use Chrome (non-incognito) it looks like this:
<img width="1627" alt="Screen Shot 2022-08-16 at 7 08 56 PM" src="https://user-images.githubusercontent.com/7035560/184949156-485e36b7-8559-4c7d-bc75-a84e1bd38818.png">

It works fine in safari, firefox, and chrome incognito.

**Reply by slorber** [reply]
then the problem is related to your browser (may be a plugin extension, cache or whatever)

we can't do anything, if it works in chrome incognito, then it works
